### PR TITLE
Fix double total stats reported in table simple output

### PR DESCRIPTION
**Pull request by roman-kruglov**
# Description

When run with simple output format the table reported doubled total stats. There was an additional loop for totals which I didn't notice when copied table's formatter implementation. Also I shrank the table implementation a bit.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

By running in a terminal

## Test Configuration

- Release version: latest dev (current release version is .33)
- Platform: Mac OS

### CODE REVIEW

File: src/utils/metrics/reporter.go [hunk 1]
```diff
@@ -44,37 +44,33 @@ func NewConsoleReporter(target io.Writer) Reporter {
...
-		fmt.Fprintf(w, "|\t%s\t|\t%d\t|\t%d\t|\t%d\t|\t%.2f MB \t|\n", tgt,
-			tgtStats[RequestsAttemptedStat],
-			tgtStats[RequestsSentStat],
-			tgtStats[ResponsesReceivedStat],
-			float64(tgtStats[BytesSentStat])/BytesInMegabyte,
-		)
-
-		for s := range totals {
-			totals[s] += tgtStats[s]
-		}
```
Comment (roman-kruglov) [inline src/utils/metrics/reporter.go:70]: this was the actual loop which doubled the total stats - now all calculations are done by `stats, totals := metrics.SumAllStats()` call and there is no need for reporters to calculate anything

File: src/utils/metrics/reporter.go [hunk 2]
```diff
@@ -44,37 +44,33 @@ func NewConsoleReporter(target io.Writer) Reporter {
...
-	w := tabwriter.NewWriter(r.target, 1, 1, 1, ' ', tabwriter.AlignRight)
-
-	defer w.Flush()
-
+	writer := tabwriter.NewWriter(r.target, 1, 1, 1, ' ', tabwriter.AlignRight)
 	stats, totals := metrics.SumAllStats()
 
-	fmt.Fprintln(w, "\n --- Traffic stats ---")
-	fmt.Fprintf(w, "|\tTarget\t|\tRequests attempted\t|\tRequests sent\t|\tResponses received\t|\tData sent \t|\n")
+	defer writer.Flush()
```
Comment (Arriven) [inline src/utils/metrics/reporter.go:50]: why did you move flush after SumAllStats()?

Comment (roman-kruglov) [reply src/utils/metrics/reporter.go:50]: just to combine variable declarations together

**Review by Arriven — APPROVED** [review]
